zipline/testing/slippage.py

```diff
@@ -0,0 +1,32 @@
+from zipline.finance.slippage import SlippageModel
+from zipline.utils.sentinel import sentinel
+
+
+class TestingSlippage(SlippageModel):
+    """
+    Slippage model that fills a constant number of shares per tick, for
+    testing purposes.
+
+    Parameters
+    ----------
+    filled_per_tick : int or TestingSlippage.ALL
+        The number of shares to fill on each call to process_order. If
+        TestingSlippage.ALL is passed, the entire order is filled.
+
+    See also
+    --------
+    zipline.finance.slippage.SlippageModel
+    """
+    ALL = sentinel('All')
```

Comment:
In most other places, we use the same name for the sentinel value and the string we pass for its repr.  Any reason not to do that here as well?